CONTRIBUTING.md

```diff
@@ -1,37 +1,99 @@
-# Contributions
+# Contributing to Netlify Essential Plugins
 
-🎉 Thanks for considering contributing to this project! 🎉
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
+
+The following is a set of guidelines for contributing to Netlify and its packages, which are hosted in the Netlify Organization on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+Please note, we have a [code of conduct](CODE_OF_CONDUCT.md). Please follow it in all your interactions with this project.
 
 These guidelines will help you send a pull request.
 
-If you're submitting an issue instead, please skip this document.
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Contributor Covenant Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to the Netlify team.
+
+## I don't want to read this whole thing I just have a question!!!
+
+> **Note:** Please don't file an issue to ask a question. You'll get faster results by using the resources below.
```

Comment:
imo if it's a plugin-specific question, i'm not sure it's fair to direct them to the community forum because itll just end up coming back here to us. personally id remove this section